### DISCUSSION TITLE
Add shortcut for --merge-input-documents

### DISCUSSION THREAD

**Discussion by burtgulash** [TomWright/dasel#170] [Ideas]
It would be a great to have a fast cli tool to convert from jsonlines to csv, which can be done using dasel as

```
dasel -r json -w csv --merge-input-documents
```

It would be better, if this could be done with a shortcut, such as

```
dasel -r json -w csv -j
```

-m is already taken for --multiple. -j could be used to stand for 'join', or -a for 'all'.

**Comment by TomWright** [comment]
I had purposely left out a shortcut for this as it's usage it's quite specific and to my knowledge not commonly used.

If my understanding is wrong then I'm happy to reconsider but until then I'd prefer not to add a shortcut as I don't want to pollute the namespace with uncommon/convoluted shortcuts that don't match the long-form flag name.

What happens if a `merge-output-documents` is added? `-m`, `-o` and `-j` will all be used then. `-a` for all could have mixed meanings and doesn't really describe what it's doing.

If there are any other ideas around this then please raise them, but I'm not going to rush into a decision here because it will lock dasel into this decision for the foreseeable future.
